README.md

```diff
@@ -2,19 +2,20 @@
 
 ## Docker
 
+Run this command to start SonarQube server.
+
 > docker run -d --name sonarqube -e SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true -p 9000:9000 sonarqube:latest
 
-## Discord Bot token
+## Add your secrets
 
-To make this work with your discord-bot you need to add your bot-token to an ***application.properties*** file inside
-the resources folder:
+Create a file named `secret.properties` in `src/main/resources`. The following secrets are required to run the
+application.
 
-> token=botToken123
+### Discord Bot
 
-## Sonarqube server
+> token=botToken123
 
-***application.properties*** file inside the resources folder:
+### Sonarqube server
 
-> sonarqubeAPI=port/api<br>

```

Comment:
I separated public and secret application properties.
This property is in the public file and I don't think we need to explain those.